recipes/spacetrack/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spacetrack" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/spacetrack-{{ version }}.tar.gz
+  sha256: fe4b54c3dede0496a0ecc4039211481a25ae4fbd0eff5e54190441f0d4434c74
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=61
+    - pip
+  run:
+    - python >={{ python_min }}
+    - filelock >=3.17.0
+    - httpx >=0.28.1
+    - logbook >=1.8.0
+    - outcome >=1.3.0.post0
+    - platformdirs >=4.3.6
+    - python-dateutil >=2.9.0.post0  # [py<311]
+    - represent >=2.1
+    - rush >=2021.4.0
+    - sniffio >=1.3.1
+
+test:
+  imports:
+    - spacetrack
+  commands:
+    - pip check
+  requires:
+    - pip
+    - python {{ python_min }}
+
+about:
+  home: https://pypi.org/project/spacetrack/
+  summary: Python client for space-track.org
+  doc_url: https://spacetrack.readthedocs.io
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - RazerM
```

Comment:
Please remove me from here, I didn't create this recipe nor do I wish to maintain it.